cloud/amazon/rds.py

```diff
@@ -647,8 +652,9 @@ def get_data(self):
 
 def await_resource(conn, resource, status, module):
     wait_timeout = module.params.get('wait_timeout') + time.time()
+    api_throttling = module.params.get('api_throttling')
```

Comment:
Instead of manually doing retries, use [AWSRetry](https://github.com/ansible/ansible-modules-core/blob/db66833a80bcd7f0d33ccbfd6555d69c5ae88003/cloud/amazon/cloudformation.py#L317) which is now part of the EC2 module_utils. It got added early in 2.3, and should cut down on duplicated retry/backoff logic.